documentation/man/kw.rst

```diff
@@ -349,6 +349,21 @@ displays the current month data.
 4. *--year [YEAR]*: exhibits the current year summary if the user does not
 specify a year.
 
+p, pomodoro --set-time|-t INTEGER[h|m|s] | --current
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This feature provides a technique to help kernel developers to manage their
+time through the Pomodoro technique. If a user wants to set a timer to provide
+visual and sonorous sound after a few minutes, they can use:
+
+1. *--set-time,-t INTEGER{h|m|s}*: Users can specify a timer that will trigger
+an alert after X (where X is a positive INTEGER number), hours (h), minutes
+(m), and seconds (s). It is mandatory to define the time metric as a suffix of
+the number.
+
```

Comment:
I think this bit could use some clarification, the timer takes hours, minutes and seconds as parameters all at the same time? for example, can I set a timer for 4 hours and 35 seconds or you have to choose a single time unit?

Reply:
@rodrigosiqueira I think it is clearer now, but I still don´t understand if we can  mix units haha can we do something like 1h 30m ?

Reply:
Hi @alan-barzilay ,
I'm not sure how users can think that we might accept something like 1h 30m if the syntax is:
`--set-timer|-t INTEGER[h|m|s]`
In summary, you cannot mix units. What is your suggestion to remove this ambiguity?

Reply:
I think I misread the syntax, I will mark this as resolved since I agree with you and don't have an issue with this anymore